parent/pom.xml

```diff
@@ -2103,6 +2133,71 @@
                </plugin>
 
                <plugin>
+                  <groupId>org.codehaus.mojo</groupId>
+                  <artifactId>build-helper-maven-plugin</artifactId>
+                  <version>1.7</version>
+                  <executions>
+                     <execution>
+                        <id>add-source</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                           <goal>add-source</goal>
+                        </goals>
+                        <configuration>
+                           <sources>
+                              <source>${dir.ispn}/atomic-factory/src/main/java</source>
+                              <source>${dir.ispn}/cdi/common/src/main/java</source>
+                              <source>${dir.ispn}/cdi/embedded/src/main/java</source>
+                              <source>${dir.ispn}/cdi/remote/src/main/java</source>
+                              <source>${dir.ispn}/checkstyle/src/main/java</source>
+                              <source>${dir.ispn}/cli/cli-client/src/main/java</source>
+                              <source>${dir.ispn}/cli/cli-interpreter/src/main/java</source>
+                              <source>${dir.ispn}/client/hotrod-client/src/main/java</source>
```

Comment:
Could you tell me why do we need this part?

It might be really hard to maintain it properly... I bet someone will create a new module and forget to add it here... IMO we need some other approach